cmd/snap/cmd_snap_op.go

```diff
@@ -872,6 +892,81 @@ func (x *cmdRefresh) Execute([]string) error {
 	return x.refreshMany(names, opts)
 }
 
+func (x *cmdRefresh) holdRefreshes() (err error) {
+	var opts client.SnapOptions
+
+	if x.Hold == "forever" {
+		opts.Time = "forever"
+	} else {
+		dur, err := time.ParseDuration(x.Hold)
+		if err != nil {
+			return fmt.Errorf("hold value must be a number of hours or minutes (e.g., 72h): %v", err)
+		}
+
+		opts.Time = timeNow().Add(dur).Format(time.RFC3339)
+	}
+
+	names := installedSnapNames(x.Positional.Snaps)
+	var changeID, snapStr string
+	if len(names) == 1 {
+		changeID, err = x.client.HoldRefreshes(names[0], &opts)
+		snapStr = names[0]
+	} else {
+		changeID, err = x.client.HoldRefreshesMany(names, &opts)
+		if len(names) == 0 {
+			snapStr = "all snaps"
+		} else {
+			snapStr = strutil.Quoted(names)
+		}
+	}
+
+	if err != nil {
+		return err
+	}
+
+	_, err = x.wait(changeID)
+	if err != nil {
+		if err == noWait {
+			return nil
+		}
+		return err
+	}
+
+	fmt.Fprintf(Stdout, i18n.G("Held refreshes of %s until %s\n"), snapStr, opts.Time)
+	return nil
+}
+
+func (x *cmdRefresh) unholdRefreshes() (err error) {
+	names := installedSnapNames(x.Positional.Snaps)
+	var changeID, snapStr string
+	if len(names) == 1 {
+		changeID, err = x.client.UnholdRefreshes(names[0], nil)
+		snapStr = names[0]
+	} else {
+		changeID, err = x.client.UnholdRefreshesMany(names, nil)
+		if len(names) == 0 {
+			snapStr = "all snaps"
+		} else {
+			snapStr = strutil.Quoted(names)
+		}
+	}
+
+	if err != nil {
+		return err
+	}
+
+	_, err = x.wait(changeID)
+	if err != nil {
+		if err == noWait {
+			return nil
+		}
+		return err
+	}
+
+	fmt.Fprintf(Stdout, i18n.G("Removed hold on refreshes for %s\n"), snapStr)
```

Comment:
same here:

"... hold on auto-refreshes of all snaps..."
"... hold on general refreshes for %s..."